### PR TITLE
Docs updates for new command format for v1.9

### PR DESCRIPTION
**Pull request by Tryneus**
Sister PR for #3617 - this updates all places referencing the changed commands (that I could find).

### CODE REVIEW

File: doc/fundamentals/getting_data_into_pachyderm.md [hunk 1]
```diff
@@ -48,67 +48,67 @@ To get data into Pachyderm using `pachctl`, you first need to create one or
...
-$ pachctl create-repo <repo name>
+$ pachctl create repo <repo name>
 ```
 
-Then, to put data into the created repo, you use the `put-file` command. Below
-are a few example uses of `put-file`, but you can see the complete
-documentation [here](../pachctl/pachctl_put-file.html).
+Then, to put data into the created repo, you use the `put file` command. Below
+are a few example uses of `put file`, but you can see the complete
+documentation [here](../pachctl/pachctl_put_file.html).
```
[the remark this answers is not in the document]
Comment (Tryneus) [reply doc/fundamentals/getting_data_into_pachyderm.md]: Yep!

File: doc/getting_started/beginner_tutorial.rst [hunk 2]
```diff
@@ -36,34 +36,34 @@ Now that we've created a repo it's time to add some data. In Pachyderm, you writ
...
 
-Unlike Git, commits in Pachyderm must be explicitly started and finished as they can contain huge amounts of data and we don't want that much "dirty" data hanging around in an unpersisted state. `Put-file` automatically starts and finishes a commit for you so you can add files more easily. If you want to add many files over a period of time, you can do `start-commit` and `finish-commit` yourself.
+Unlike Git, commits in Pachyderm must be explicitly started and finished as they can contain huge amounts of data and we don't want that much "dirty" data hanging around in an unpersisted state. ``put file`` automatically starts and finishes a commit for you so you can add files more easily. If you want to add many files over a period of time, you can do ``start commit`` and ``finish commit`` yourself.
 
 We also specify the repo name "images", the branch name "master", and the file name: "liberty.png".
 
-Here's an example atomic commit of the file `liberty.png` to the `images` repo's `master` branch:
+Here's an example atomic commit of the file ``liberty.png`` to the ``images`` repo's ``master`` branch:
```
[the remark this answers is not in the document]
Comment (Tryneus) [reply doc/getting_started/beginner_tutorial.rst]: For the record, I believe `.rst` files use a different flavor of markup, so it's double-backticks for code in a `.rst` file.

File: examples/scraper/README.md [hunk 3]
```diff
@@ -224,26 +224,26 @@ more data to the same file “urls.”
 Let's create a new commit with our previous commit as the parent:
 
 ```shell
-$ pachctl start-commit urls master
+$ pachctl start commit urls@master
 master/1
```
[the remark this answers is not in the document]
Comment (Tryneus) [reply examples/scraper/README.md]: How should I deprecate it?

[the remark this answers is not in the document]
Comment (JoeyZwicker) [reply examples/scraper/README.md]: good question. It actually looks like in the README we already have a note there saying this demo is from 1.4. Can you just make that message much more obvvious --bolder/highlighted or something and maybe put a "depricated" label in the readme, that'll be good enough.

**Pull request merged**